srcpkgs/OpenJK/files/openjk

```diff
@@ -0,0 +1,9 @@
+#! /bin/sh
+OJ_ARCH=$(uname -m)
+case "$OJ_ARCH" in
+  arm*) OJ_ARCH=arm;;
+  i*86*) OJ_ARCH=i386;;
+esac
+
+cd /usr/lib/OpenJK/JediAcademy
+exec ./openjk.${OJ_ARCH} $@
```

Comment:
you forgot to quote it, read up on shell semantics for $@ with double quotes vs not

Reply:
ah, I was not aware that it is basically $* without the quotes. thank you for the note :)